### ISSUE TITLE
Polish up drag-and-drop

### ISSUE THREAD

**Issue by dethe** [waterbearlang/tardigrade#6]
* [x] Cannot drag to the interior of a context or trigger
* [x] Cannot drag from locals
* [x] Cannot drag from returns
* [ ] Dragging a value to the script does not transform to a step
* [x] Cannot drag a value to a param

**Comment by dethe** [comment]
Update: Currently **all** dragging is broken because `dragging.js` still uses the `wb-` prefixes instead of `tg-` prefixes for elements (due to resolving #5). That part should be easy to fix, but needs to be done first.

**Comment by dethe** [comment]
I'm going to leave the resolution of value vs. step to another issue. Also enforcing correct types in sockets.
